### ISSUE TITLE
Noise scheduler bug

### ISSUE THREAD

**Issue by lawrence-cj** [city96/ComfyUI_ExtraModels#3]
Hi there, thank you for your implementation of PixArt-Alpha on ComfyUI. It's a big step.
I found a little bug in the code causing poor image generation.
We may fixed it together.

Specifically, Line 41: [betas = torch.tensor(gd.get_named_beta_schedule(noise_schedule, steps)) ](https://github.com/city96/ComfyUI_ExtraModels/blob/389c16f2f51b931f758169e1e4c0b933fdb95244/PixArt/sampler.py#L41C26-L41C26) needs to be changed to:

`betas = torch.tensor(gd.get_named_beta_schedule(noise_schedule, 1000))` in the most simple way.

**Comment by city96** [comment]
Good catch, thanks!

Seems like you're right and that argument [should be left as the default of 1000](https://github.com/PixArt-alpha/PixArt-alpha/blob/06611964f2782cf31d9075a43697f5e5d30499e8/diffusion/dpm_solver.py#L15).

That reference sampler code was meant to be a stopgap until I got the model to work with the default KSampler, but I never figured out why it performed so poorly with it. Tried messing with the SamplerCustom node but couldn't get it to match the reference code no matter how much I messed with it...

![Untitled](https://github.com/city96/ComfyUI_ExtraModels/assets/125218114/fcb2963b-ee1f-46c4-98e7-4d42a8a719e7)
